Source/JavaScriptCore/tools/Integrity.h

```diff
@@ -98,6 +98,10 @@ class Random {
 ALWAYS_INLINE static bool isSanePointer(const void* pointer)
 {
     uintptr_t pointerAsInt = bitwise_cast<uintptr_t>(pointer);
+#if CPU(ARM64)
+    // On ARM64, top byte ignore means we can ignore these bits for addresses.
+    pointerAsInt &= (1ull << 56) - 1;
```

Comment:
Nit: `std::numeric_limits<uintptr_t>::max() >> CHAR_BIT` is clearer to me but either way is fine.

Reply:
Resolved in latest push.